packages/data/src/world/mq/deku_tree_mq.yml

```diff
@@ -70,6 +70,8 @@
   exits:
     "Deku Tree Lobby": "true"
     "Deku Tree Water Room Back": "(is_child && has_shield) || can_longshot || (can_hookshot && has_iron_boots)"
+  events:
+    MQ_DEKU_WATER_TORCHES: "(is_child && has_shield && can_use_sticks) || (can_use_bow && can_play_elegy)"
```

Comment:
Does this need `|| has_fire` like was removed from line 81?

Reply:
Oh nevermind.

Reply:
Could fire arrow matter here, if you don't have elegy?

Reply:
all it does is open the door on the other side of the room, so no